x-pack/plugins/security_solution/scripts/endpoint/action_responder/utils.ts

```diff
@@ -114,3 +117,16 @@ export const sendEndpointActionResponse = async (
 
   return endpointResponse;
 };
+
+const getOutputDataIfNeeded = (command: EndpointActionData['command']) => {
+  return command === 'running-processes'
+    ? ({
+        output: {
+          type: 'json',
+          content: {
+            entries: endpointActionGenerator.randomResponseActionRunningProcesses(),
```

Comment:
suggestion: use a number like 50 or even 100 here when calling `randomResponseActionRunningProcesses()` so that we get more realistic data back. Maybe ever sett it to a higher number. We will want to show Meril and others how the possible large amount of data will be shown in the UI when the response is returned.